keras_eval/eval.py

```diff
@@ -102,7 +105,8 @@ def _get_complete_image_paths(self, filenames):
         return image_paths
 
     def evaluate(self, data_dir=None, top_k=1, filter_indices=None, confusion_matrix=False,
-                 save_confusion_matrix_path=None):
+                 save_confusion_matrix_path=None, combine_training_classes=False, combine_classes_dict_dir=None,
```

Comment:
I will check for errors in the dictionary but I don't think it is necessary for the user to provide an `id`, we assume that the order they write the file in is the order they want. Also, I think it will be more readable with `sub_class` name as when anyone looks at the `.json` will understand which classes are being grouped together.